ads/google/a4a/traffic-experiments.js

```diff
@@ -165,9 +173,24 @@ function maybeSetExperimentFromUrl(win, element, experimentName,
     '0': null, // TODO Ensure does not generate exp id
     '1': controlBranchId,
     '2': treatmentBranchId,
+    '3': hasLaunched(win, element) ? treatmentBranchId : controlBranchId, // DF
   };
   if (argMapping.hasOwnProperty(arg)) {
     forceExperimentBranch(win, experimentName, argMapping[arg]);
+    if (arg == '3') {
```

Comment:
forceExperimentBranch is given argMapping so I'm not sure why you need this special behavior?  So long as 3 is given a different experiment ID than existing control/experiment and is treated as if control (just sends a different eid in the ad request), we should be ok.  We can then control delayed impression usage for that id server side